### PR TITLE
Fix dbt env platform

### PR DESCRIPTION
**Pull request by elic-eon**
### 🤔 Why?

Can not directly map the adapter type to `DataPlatform`

### 🤓 What?

- Add a dictionary for the conversion

### 🧪 Tested?

Run the crawler against an Postgres project in dbt cloud

### ☑️ Checks

- [x] My PR contains actual code changes, and I have updated the version number in `pyproject.toml`.

### CODE REVIEW

File: metaphor/dbt/cloud/utils.py [hunk 1]
```diff
@@ -11,6 +11,8 @@ def parse_environment(environment: GetEnvironmentAdapterTypeEnvironment):
     adapter_type = adapter_type.upper()
     if adapter_type == "DATABRICKS":
         platform = DataPlatform.UNITY_CATALOG
+    elif adapter_type == "POSTGRES":
```
Comment (mars-lan) [inline metaphor/dbt/cloud/utils.py:14]: Add a dict?

Comment (mars-lan) [reply metaphor/dbt/cloud/utils.py:14]: And include all relevant types?

Comment (elic-eon) [reply metaphor/dbt/cloud/utils.py]: Added

File: metaphor/dbt/cloud/utils.py [hunk 2]
```diff
@@ -3,14 +3,19 @@
 )
 from metaphor.models.metadata_change_event import DataPlatform
 
+ADAPTER_TYPE_MAPPING = {
+    "DATABRICKS": DataPlatform.UNITY_CATALOG,
+    "POSTGRES": DataPlatform.POSTGRESQL,
```
Comment (mars-lan) [inline metaphor/dbt/cloud/utils.py:8]: Consider adding other known cases, instead of relying on enum conversations?

Comment (elic-eon) [reply metaphor/dbt/cloud/utils.py:8]: but we have not test other case

Comment (mars-lan) [reply metaphor/dbt/cloud/utils.py:8]: Not even Snowflake?

Comment (elic-eon) [reply metaphor/dbt/cloud/utils.py:8]: My bad, we have "SNOWFLAKE", added

**Review by mars-lan — APPROVED** [review]

**Pull request merged**